docs/contribute.rst

```diff
@@ -324,25 +346,54 @@ whether given functionality is already available in their package:
 
     .. versionadded:: 1.23
 
-If the pull request addresses an existing issue, mention it using
-one of the automatically parsed formats so that it is linked to
-it, for example:
+.. _review:
 
-.. code-block:: markdown
+Review
+------------------------------------------------------------------
 
-    Fix #1234.
+Code review is an essential part of the workflow. It ensures good
+quality of the code and prevents introducing regressions, but it
+also brings some additional benefits: By reading code written by
+others you can learn new stuff and get inspired for your own code.
+Each completed pull request review helps you, little by little, to
+get familar with larger part of the project code and empowers you
+to contribute more easily in the future.
 
-By default only a core set of tests is executed against a newly
-created pull request and its updates to verify basic sanity of the
-change. Once the pull request content is ready for a thorough
-testing add the ``full test`` label and make sure that the
-``discuss`` label is not present. All future changes of the pull
-request will be tested with the full test coverage. For changes
-related to documentation only the full test suite is not required.
+For instructions how to locally try a change on your laptop see
+the :ref:`develop` section. Basically just enable the development
+environment and check out the pull request branch or use the
+`github cli`__ to check out code from a fork repository:
+
+.. code-block:: shell
+
+    hatch -e dev shell         # enable the dev environment
+    git checkout the-feature   # if branch is in the tmt repo
+    gh pr checkout 1234        # check out branch from a fork
+
+Note that you don't have to always read the whole change. There
+are several ways how to provide feedback on the pull request:
+
+* check the **documentation**, look for typos, identify wording
```

Comment:
How about mentioning how to open a pr-specific RTD builds? Like for example this one - https://tmt--2886.org.readthedocs.build/en/2886/contribute.html#review

Reply:
Good idea! Added in e62c0008.

Reply:
Hmm, I'm not sure this is clear on how to actually open the rendered docs.  
I would normally open https://readthedocs.org/projects/tmt/builds/ and find the PR number, but don't know if that's the correct way to open it.

Reply:
That would be the "Details" link of the `docs/readthedocs.org:tmt` check below. If you can see it, of course, I think sometimes it's hidden from some folks :/

Reply:
Right, for me it shows below, but when I searched for it in `Checks` tab, it's not there.  
Maybe it's just me, but I would expect people to be confused.  

For dumb people like me, maybe mention the "Details" link and "docs/readthedocs.org:tmt" as a name of a workflow check, not `docs/readthedocs.org` code-formatted which looks like and address or a path.

But it's fine as is, just a nitpick from an autist.

Reply:
Yeah, for some reason the docs check only appears at the bottom of the main pull request page but not on the `Checks` tab. However, I don't think it would be good to make the description any longer / more complex, there will always be situations when one has to ask for the hint on the chat, I guess this would one of them ;-)